<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Base64</string>
     </attribute>
     <widget class="QPushButton" name="base64EncodeBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Base64 encode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="base64DecodeBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Base64 decode</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="base64DecodeInput">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="base64DecodeOutputLbl">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="base64EncodeOutput">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="base64EncodeInput">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="hexEncodeBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Hex encode</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hexEncodeInput">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="hexDecodeBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Hex decode</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hexDecodeInput">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>AES</string>
     </attribute>
     <widget class="QPushButton" name="aesEncryptBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>AES encrypt</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="aesEncryptInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="aesDecryptInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="aesDecryptBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>AES decrypt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="aesGenKeyBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Key</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="aesGenKeyInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="aesGenIvBlockInput">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>RSA</string>
     </attribute>
     <widget class="QLineEdit" name="rsaPrivateKeyInput">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Private key</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Public Key</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rsaPublicKeyInput">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rsaEncryptPubKeyBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt w/ pub key</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Plain text</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cipher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rsaDecryptPrivKeyBtn">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt w/ priv key</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateKpBtn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate KP</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="rsaPlainTextInput">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="rsaEncryptedTextInput">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
